src/Sylius/Bundle/AdminBundle/index.js

```
@@ -11,20 +11,35 @@ const path = require('path');
 const Encore = require('@symfony/webpack-encore');
 
 class SyliusAdmin {
+    /**
+     * Provide a light Webpack configuration for Sylius Admin
+     * All the stimulus stuff should be handled by the app.admin entrypoint
+     */
+    static getBaseWebpackConfig(rootDir) {
+        this._prepareWebpackConfig(rootDir);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)